data/data-pipeline/data_pipeline/tests/sources/national_risk_index/test_etl.py

```diff
@@ -91,9 +92,10 @@ def test_load(self, mock_etl):
         expected = pd.read_csv(DATA_DIR / "output.csv", dtype={BLOCK_COL: str})
         # execution
         etl.load()
-        output = pd.read_csv(output_path, dtype={BLOCK_COL: str})
 
+        output = pd.read_csv(output_path, dtype={BLOCK_COL: str})
```

Comment:
i think this `BLOCK_COL` should be replaced everywhere with the census tract column instead. 

That is on me though! I'm the one who didn't update these tests enough when switching to tracts.